crates/beerus-core/src/lightclient/beerus.rs

```diff
@@ -126,32 +126,13 @@ impl BeerusLightClient {
             self.starknet_lightclient.start().await?;
             self.sync_status = SyncStatus::Synced;
 
-            let ethereum_clone = self.ethereum_lightclient.clone();
             let starknet_clone = self.starknet_lightclient.clone();
             let node_clone = self.node.clone();
             let poll_interval_secs = self.config.get_poll_interval();
 
             // Define function that will loop
             let task = async move {
                 loop {
-                    let state_root = ethereum_clone
```

Comment:
These are locking threads and starving the app when run on low cpu environments.